config/locales/de.yml

```diff
@@ -45,14 +45,40 @@ de:
     oauth_text: "Dieses Konto ist durch %{provider} authentifiziert."
     own_projects: "Von dir gestartete Projekte"
     participating_projects: "Projekte, an denen du teilnimmst"
+    projects: "Deine Projekte"
+    no_projects: "Es gibt keine Projekte, an denen du bereits teilnimmst, oder die du veranstaltest."
+    all_projects_pre: "Alle vorhandenen Projekte"
+    show: "anzeigen"
+    all_projects_post: ""
+    new_projects_pre: "Ein neues Projekt"
+    create: "anlegen"
```

Comment:
Rails offers a much nicer alternative through the `%{...}` variable expansion feature in locale files.